app/models/enrollment.rb

```diff
@@ -0,0 +1,10 @@
+class Enrollment < ActiveRecord::Base
+  belongs_to :game
+  belongs_to :white_player_id, class_name: "Player"
+  belongs_to :black_player_id, class_name: "Player"
+
+  def join_game(game)
```

Comment:
Does the method need to store one of the players (perhaps the current_user?) I'm also thinking of the code I wrote that I creates a new game with a form where the user can select their color (black, white) -- how will this interact with that code?